lib/util/hud_utility_2024.rb

```diff
@@ -762,4 +762,23 @@ def assessment_name_by_data_collection_stage
       6 => 'Post-Exit Assessment',
     }.freeze
   end
+
+  # Utility for defining age range logic in one place.
+  # These can overlap, not all are used in every form dropdown/filter
+  def age_range
```

Comment:
Flagging this to @eanders for awareness. I couldn't find an existing mapping like this. These values were defined in an HMIS GQL Enum that we used for the Age Range table filter on client/enrollment tables in HMIS. Now (some of them) are also being used for PIT age categories, and this PR introduces logic for processing age range responses into low-data-quality DOB values.

Reply:
Thanks, I saw this.  I think this is probably fine.  Eventually we should build this out to support/share code with:
https://github.com/greenriver/hmis-warehouse/blob/f039ec719577065e8d3c7c0987b0ee3e3a131c40/app/models/concerns/filter/filter_scopes.rb#L84-L103
and
https://github.com/greenriver/hmis-warehouse/blob/f039ec719577065e8d3c7c0987b0ee3e3a131c40/app/models/filters/filter_base.rb#L907-L1008

Those are the two that I know we've been trying to standardize on.  There are some additional ranges specific to HUD reports here
https://github.com/greenriver/hmis-warehouse/blob/f039ec719577065e8d3c7c0987b0ee3e3a131c40/app/models/concerns/hud_reports/ages.rb#L51-L64
and some I don't have any memory of here:
https://github.com/greenriver/hmis-warehouse/blob/f039ec719577065e8d3c7c0987b0ee3e3a131c40/app/models/concerns/known_categories/age.rb#L10-L33